sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,328 @@
+
+## New features
+
+### SYCL Compiler
+- Enabled Pre-Compiled header (PCH) inclusion while performing host compilation with `-fsycl`. [7f12b287236f]
+- Turned on SPV extension `SPV_INTEL_fpga_latency_control` in the clang driver.  [1aabcf8eb9ed]
+- Added prototype implementation of the Device config file designed in [`DeviceConfigFile`](doc/design/DeviceConfigFile.md) [0bb722138216]
+- Added support for `-ftarget-export-symbols` option which allows to export symbols for AOT. [c33ba5dafe91]
+- Added support of `reqd_sub_group_size` to `sycl::is_compatible` and implemented device code split based on `reqd-sub-group-size` optional kernel feature. [841768794f7a]
+- Added support for `-ftarget-compile-fast` in JIT mode. [92855bea87f4]
+- Implemented adding C++ libraries to the linker with `-fsycl`. [da5e1b98f1c3]
+- Added support for `-ftarget-compile-fast`. [b6b01b48cfd9]
+- Added support for multiple call operators in a kernel functor. [84cc1e1eff04]
+- Added support to propagate compile flags to device backend compiler.[f45fb51ba6cf]
+- Enabled `SPV_INTEL_fpga_invocation_pipelining_attributes` spirv extension in the DPCPP compiler. [3bc05248602a]
+
+### SYCL Library
+- Enabled [`sycl_ext_oneapi_annotated_arg`](doc/extensions/experimental/sycl_ext_oneapi_annotated_arg.asciidoc) and [`sycl_ext_oneapi_annotated_ptr``](doc/extensions/experimental/sycl_ext_oneapi_annotated_ptr.asciidoc)[80b7de1d63f4] experimental extensions. [05d3be63f996][80b7de1d63f4][557204abf242]
+- Implemented [`sycl_ext_intel_queue_immediate_command_list`](doc/extensions/supported/sycl_ext_intel_queue_immediate_command_list.asciidoc) extension. [2f2560fe74e6]
+- Implemented [`sycl_ext_oneapi_copy_optimize`](sycl/doc/extensions/experimental/sycl_ext_oneapi_copy_optimize.asciidoc) experimental extension. [65cc0cfe809f]
+- Added initial implementation of the experimental SYCL Graph extension [`sycl_ext_oneapi_graph`](doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc)  [35465da4824a][f9d4830babca][72341ee375d6][96a605095665]
+- Added `dimensions` member to item/range-like types. [9c7bd9ea046f]
+- Added support of `queue::priority_*` properties for OpenCL backend. [25fd689999f7]
+- Implemented initial version of the SYCL Native CPU Plug-in designed in [SYCLNativeCPU](doc/design/SYCLNativeCPU.md). [b2d0837e9644] [5b501eeca963]
+- Added USM calls parameter verification layer to `sycl-trace` which provides improved diagnostic on accessing invalid device memory under USM. [758bd264bb90]
+- Added `__imf_max/min/hadd/fast_*` functions to imf device libraries. [fe01366e8b08][d96e5075303a]
+- Introduced and implemented new `sycl::ext::oneapi::experimental::info::device::architecture` device descriptor and `device::ext_oneapi_architecture_is(ext::oneapi::experimental::architecture)` host API as part of [`sycl_ext_oneapi_device_architecture`](doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc)  extension (for Level Zero and OpenCL). [d0b01b265d92]
```

Comment:
Also for CUDA and HIP: https://github.com/intel/llvm/pull/10573

Reply:
It looks like CUDA and HIP support was committed after the range of commits covered by these release notes (Mar 17 - Jul 24). As far as I understand it's better to mention this in the next release notes.